src/lib/controller/message_handler.ts

```diff
@@ -191,55 +191,57 @@ function processInclusionOptions(
       let validateDSKAndEnterPinPromise:
         | DeferredPromise<string | false>
         | undefined;
-      options.userCallbacks = {
-        grantSecurityClasses: (
-          requested: InclusionGrant
-        ): Promise<InclusionGrant | false> => {
-          clientsController.grantSecurityClassesPromise =
-            grantSecurityClassesPromise = createDeferredPromise();
-          grantSecurityClassesPromise.finally(() => {
-            if (
-              clientsController.grantSecurityClassesPromise ===
-              grantSecurityClassesPromise
-            ) {
-              delete clientsController.grantSecurityClassesPromise;
-            }
-          });
-          client.sendEvent({
-            source: "controller",
-            event: "grant security classes",
-            requested: requested as any,
-          });
+      if (!("provisioning" in options)) {
```

Comment:
Are these changes intended?

Reply:
yes, this is required, otherwise the package won't build due to an addiitonal options schema that was added